lldb/source/Plugins/Process/gdb-remote/GDBRemoteCommunicationServerLLGS.cpp

```diff
@@ -1140,6 +1140,14 @@ void GDBRemoteCommunicationServerLLGS::HandleInferiorState_Exited(
   Log *log = GetLog(LLDBLog::Process);
   LLDB_LOGF(log, "GDBRemoteCommunicationServerLLGS::%s called", __FUNCTION__);
 
+  // Notify GPU plugins that the native process has exited
+  auto exit_status = process->GetExitStatus();
+  if (exit_status.has_value()) {
+    for (auto &plugin_up : m_plugins) {
+      plugin_up->NativeProcessDidExit(*exit_status);
+    }
+  }
```

Comment:
don't use auto unless the type is unreadable. In this case it should be very readable